app/controllers/activity_insight_oa_workflow/wrong_file_version_curation_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class ActivityInsightOAWorkflow::WrongFileVersionCurationController < ActivityInsightOAWorkflowController
+  def index
+    @publications = Publication.wrong_file_version.order('wrong_oa_version_notification_sent_at DESC NULLS FIRST')
+  end
+
+  def email_author(publications)
```

Comment:
It's not clear to me how a collection of publications is getting passed in here. Our code doesn't call the controller method directly, the router does, and the router doesn't pass arguments to the method. I don't see an end-to-end integration test for this, so if I had to guess, I'd bet that clicking an email button is sending the email without any publication data.

However, it looks like the form (button) for sending the email is being constructed such that it's posting the publications as form data. That's what we want, but that data won't be passed into the controller method as an argument (and definitely not as fully-formed ActiveRecord objects). Instead, I think we'll need to pull the publication IDs out of the params hash and then find the records. Something like this:

```ruby
publications = Publication.wrong_file_version.find(params[:publications])
```

I'm also not totally sure whether or not the way in which the publications are being passed to `button_to` as params will cause the collection of publication objects to be transformed into an array of publication IDs, but that's what we'd need.